test/unit/spec/EnrollChoices_spec.js

```diff
@@ -689,232 +716,319 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, FactorUtil, R
+            singleFactorRes: singleFactorResponse,
+            webauthnEnabled: webauthnEnabled,
+            useProfiles: useProfiles
+          }).then(function (test) {
+            //enrolled factor row should have pending check and cardinality text
+            expect(test.form.factorHasPendingCheck(factorName)).toBe(true);
+            expect(test.form.factorCardinalityText(factorName)).toBe('(0 of 2 set up)');
+          });
         });
         itp('displays correct cardinality text when enrolled=1 required=1', function () {
-          return setupMultipleFactorEnrollments(factorName, 'NOT_SETUP', 'REQUIRED',
-            {'enrolled': 1, 'minimum': 2, 'maximum': 3}, singleFactorResponse)
-            .then(function (test) {
-              //enrolled factor row should have pending check and cardinality text
-              expect(test.form.factorHasPendingCheck(factorName)).toBe(true);
-              expect(test.form.factorCardinalityText(factorName)).toBe('(1 of 2 set up)');
-            });
+          return setupMultipleFactorEnrollments({
+            factorName: factorName,
+            status: 'NOT_SETUP',
+            enrollment: 'REQUIRED',
+            policyEnrollment: {'enrolled': 1, 'minimum': 2, 'maximum': 3},
+            singleFactorRes: singleFactorResponse,
+            webauthnEnabled: webauthnEnabled,
+            useProfiles: useProfiles
+          }).then(function (test) {
+            //enrolled factor row should have pending check and cardinality text
+            expect(test.form.factorHasPendingCheck(factorName)).toBe(true);
+            expect(test.form.factorCardinalityText(factorName)).toBe('(1 of 2 set up)');
+          });
         });
         itp('displays correct cardinality text when enrolled=2 required=0 optional=1', function () {
-          return setupMultipleFactorEnrollments(factorName, 'ACTIVE', 'REQUIRED',
-            {'enrolled': 2, 'minimum': 2, 'maximum': 3}, singleFactorResponse)
-            .then(function (test) {
-              var factorRows = test.form.factorRow(factorName);
-              //displays one row for enrolled, one for optional
-              expect(factorRows.length).toBe(2);
-
-              var enrolledFactorRow = factorRows[0],
-                  optionalFactorRow = factorRows[1];
-              //enrolled factor row should have success check and cardinality text
-              expect(test.form.factorHasSuccessCheck(null, enrolledFactorRow)).toBe(true);
-              expect(test.form.factorCardinalityText(null, enrolledFactorRow)).toBe('(2 set up)');
-              //optional factor should have button and no cardinality text
-              expect(test.form.factorButtonText(null, optionalFactorRow)).toBe('Set up another');
-              expect(test.form.factorCardinalityText(null, optionalFactorRow)).toBe('');
-            });
+          return setupMultipleFactorEnrollments({
+            factorName: factorName,
```

Comment:
we can use shorter notation ES6 convention here now that SIW supports it anyway,. 

factorName,
useProfiles, 
etc

Reply:
👍